package.json

```diff
@@ -83,7 +88,13 @@
     "typescript": "^2.5.3",
     "typescript-definition-tester": "^0.0.5"
   },
-  "dependencies": {},
+  "dependencies": {
+    "gitbook-cli": "^2.3.2",
```

Comment:
can you move these to dev-dependencies? otherwise they'll get installed by everyone who uses redux-loop